packages/starlight/schemas/prevNextLink.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'astro/zod';
+
+export const PrevNextLinkConfigSchema = () =>
+  z
+    .union([
+      z.boolean(),
+      z.string(),
+      z
+        .object({
+          /** The navigation link URL. */
+          href: z.string().optional(),
```

Comment:
So that would mean updating the schema here:
```suggestion
          link: z.string().optional(),
```